codacy/values-tls-ingress-aws.yaml

```diff
@@ -0,0 +1,26 @@
+## For guidance on how to set up TLS please refer to the documentation on https://codacy.github.io/chart
+
+codacy-ingress:
```

Comment:
This is in the values-production .. do we need all these values files?

Reply:
This overrides the defaults in values production. The annotations are different and this has more stuff in it. Using a different file simplifies values-production by excluding optional configuration.

Reply:
Yes, otherwise the users will (most probably) configure all the values at once in step 4 of this section of the installation instructions:

https://codacy.github.io/chart/latest/#2-installing-codacy

(But I do think that we're not being 100% consistent here, for example, the configurations for the Git providers are all in the `values-production.yaml` even though their configuration is optional since only one is required to be enabled.)

Reply:
That's a fair point. We'll need to test and document this for microk8s very shortly, so we can use that chance to try to harmonize the configuration.